docs/README.md

```diff
@@ -5,6 +5,7 @@
   * [Tutorial](/docs/tutorial/Tutorial.md)
   * [Testing](/docs/tutorial/Testing.md)
   * [Advanced topics](/docs/tutorial/AdvancedTopics.md)
+  * [Recipes](/docs/tutorial/Recipes.md)
```

Comment:
I was thinking Recipes would be a top level section like API Docs with sub pages rather than a big page under Getting started. Thoughts?

Reply:
It's not really about importance. I just don't think specific recipes fit under getting started since you would most need them not just when getting started, but if you run into a problem later on. I  also think it would just be easier to navigate around that way.

Reply:
@on3iro I think this is a dead link and needs to be removed